app/src/main/java/com/asfoundation/wallet/ui/onboarding/OnboardingActivity.kt

```diff
@@ -61,8 +64,8 @@ class OnboardingActivity : BaseActivity(), OnboardingView {
     linkSubject = PublishSubject.create()
     presenter = OnboardingPresenter(CompositeDisposable(), this, interactor,
         AndroidSchedulers.mainThread(), smsValidationInteract, Schedulers.io(),
-        ReplaySubject.create())
-    setupUi()
+        ReplaySubject.create(), referralInteractor)
+    setupUi("")
```

Comment:
Remove argument from method and remove it from the presenter. Then just call updateReferralValue from the presenter instead